physionet-django/user/enums.py

```diff
@@ -19,3 +19,12 @@ class RequiredField(IntEnum):
     @classmethod
     def choices(cls):
         return tuple((option.value, option.name) for option in cls)
+
+
+class EventCategory(IntEnum):
```

Comment:
I think a standard Enum might be better here? e.g.

```
class EventCategory(Enum):
    COURSE = "Course",
    WORKSHOP = "Workshop",
```

Even better might be to use the Django built in support for Enums. See: https://docs.djangoproject.com/en/4.1/ref/models/fields/#field-choices-enum-types

Reply:
Switched to the Django Enum in https://github.com/MIT-LCP/physionet-build/pull/1653/commits/5ce3edbc469a82031b0d093caaa3327dba139b70